docs/getting_started/building_blocks.md

```diff
@@ -50,7 +50,7 @@ These are represented in a JSON document as follows:
 
 ### Building blocks: fields
 
-The OCDS schema sets out the fields that ought to be included in each section (where applicable), making use of simple re-usable building blocks (field structures) to represent data. 
+The OCDS schema sets out fields that can be included in each section, making use of simple re-usable building blocks (field structures) to represent data. 
```

Comment:
Per my other comment, I would keep 'ought to'

Reply:
“Ought to” in combination with “where applicable” is confusing... and as weak as a “can” on its own.

Reply:
Good point, in that case I would remove 'where applicable' and keep it as 'ought to', especially since the original word was 'should'. 'where applicable' was added to try and address the issue of overfill, but that guidance/detail is probably better located in the mapping section of the implementation guidance.